tests/containers/bci_prepare.pm

```diff
@@ -81,6 +81,35 @@ sub activate_virtual_env {
     assert_script_run('source bci/bin/activate');
 }
 
+sub scc_credentials_host {
+    # Non-RMT SCC credentials for Non-suse host running BCI tests
+    my ($runtime, $scc_reg_code) = @_;
+    record_info('Credentials', "check");
+    die("Input parameter missing: $runtime, $scc_reg_code.") unless ($runtime && $scc_reg_code);
+    my $credentials = "/etc/zypp/credentials.d/SCCcredentials";
+    my $credentials_tmp = "/tmp/SCCcredentials";
+    my @commands = (
+        # Check registry connection is ok
+        "curl -ILks -m 60 -o /dev/null registry.suse.com",
+        # Collect credentials, in a local temporary file, from sle image
+        "$runtime run --rm registry.suse.com/suse/sle15:latest " .
+          "bash -c '(zypper -q -n in SUSEConnect; SUSEConnect --regcode $scc_reg_code) 2>&1 >/dev/null && " .
+          "cat $credentials' > $credentials_tmp",
+        # Validate file content
+        "grep -q 'password=' $credentials_tmp",
+        # Install credentials in SUSE standard folder
+        "sudo mkdir -p " . dirname($credentials) . "; sudo mv $credentials_tmp $credentials",
+        # Verify credentials on a bci fips image
+        "$runtime run -v $credentials:$credentials -it --pull=always registry.suse.com/bci/bci-base-fips:latest " .
```

Comment:
This introduces a new test dependency (the bci-base-fips) container here and I think this can be removed.

The objective is not to test the registration but to just enable it on non-sles hosts. I'm a bit reluctant in introducting new dependencies for non-test subjects.

Reply:
I remove it